syntax = "proto3";

package executor_profile;

option go_package = "./executor_profile";

service ExecutorProfileService {
    rpc GetExecutorProfile (ExecutorProfileRequest) returns (ExecutorProfileResponse);
}

message ExecutorProfileRequest {
    string display_name = 1;
}

message ExecutorProfileResponse {
    string id = 1;
    repeated string tags = 2;
    float rating = 3;
}
